ipl/main.c

```diff
@@ -2431,8 +2426,7 @@ ment_t ment_tools[] = {
 	MDEF_CAPTION("-------- Misc --------", 0xFF0AB9E6),
 	MDEF_HANDLER("Dump package1", dump_package1),
 	MDEF_HANDLER("Fix battery de-sync", fix_battery_desync),
-	MDEF_HANDLER("Remove archive bit (switch folder)", fix_sd_switch_attr),
-	//MDEF_HANDLER("Remove archive bit (all sd files)", fix_sd_all_attr),
```

Comment:
This one you can leave it as it is. ("Remove archive bit (all sd files)").
If you can suggest a naming to make them smaller, be my guest.

(maybe Remove can become Unset, for example )